infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/LoggingConfigurator.java

```diff
@@ -42,6 +42,7 @@ public class LoggingConfigurator {
   static final String STATUS_LOGGER_NAME = "teku-status-log";
   static final String VALIDATOR_LOGGER_NAME = "teku-validator-log";
   static final String P2P_LOGGER_NAME = "teku-p2p-log";
+  static final String DB_LOGGER_NAME = "teku-db-log";
```

Comment:
The `LoggingConfigurator` would have to be updated to enable this log and send its contents to the console same as we do for `teku-validator-log` for example.  Should be able to follow the same pattern by looking at usages of `VALIDATOR_LOGGER_NAME`.  Worth testing it works correctly in the three logging modes (file, console and both).

Reply:
Done. I tested it locally and it works (console and file and both)